### PR TITLE
Spawn eggs works again

### LINKED ISSUE #3723: Spawn eggs don't work in 1.11
Client version:   1.11.2
Server OS:        Linux
Commit id:        758622336e6c8d7aa2d4b34d8eedc4aaed54a4f6

### Expected behavior
Spawn eggs can be used in Minecraft 1.11.

### Actual behavior
Spawn eggs turn blank as soon as you put them in your inventory.

### Steps to reproduce the behavior
This is caused by a change of entity names in Minecraft 1.11. In src/Protocol/Protocol_1_9.cpp, this piece of code takes care of spawn eggs in 1.9 and above:
```
	if (a_Item.m_ItemType == E_ITEM_SPAWN_EGG)
	{
		// Convert entity ID to the name.
		eMonsterType MonsterType = cItemSpawnEggHandler::ItemDamageToMonsterType(a_Item.m_ItemDamage);
		if (MonsterType != eMonsterType::mtInvalidType)
		{
			Writer.BeginCompound("EntityTag");
			Writer.AddString("id", cMonster::MobTypeToVanillaName(MonsterType));
			Writer.EndCompound();
		}
}
```
Since MobTypeToVanillaName sends the wrong entity names in 1.11, the eggs don't work. What would be the best way to fix this issue?

### PR DESCRIPTION
**Pull request by Seadragon91**
Closes #3723.

Just to get them working.

### CODE REVIEW

**Comment by Seadragon91** [comment]
Added the constants to APIDesc and see that they have to be added to `official_undocumented.lua`.

**Comment by Seadragon91** [comment]
Should I remove them? As there are no entities implemented for them?

File: Server/Plugins/APIDump/APIDesc.lua [hunk 1]
```diff
@@ -10571,10 +10611,22 @@ a_Player:OpenWindow(Window);
...
+				},
+				mtVex =
+				{
+					Notes = "",
+				},
 				mtVillager =
 				{
 					Notes = "",
 				},
+				mtVindication_Illager =
```
Comment (madmaxoft) [inline Server/Plugins/APIDump/APIDesc.lua]: Drop the underscore, we use CamelCase everywhere else.

**Comment by Seadragon91** [comment]
Removed the new ones.

**Comment by madmaxoft** [comment]
Yea, probably easier for now.

**Pull request merged**